docker/Dockerfile-dhvirtualenv

```diff
@@ -85,6 +85,15 @@ RUN apt-get update -qq -o Acquire::Languages=none \
         libpq-dev \
         xmlsec1
 
+RUN pip3 install -U \
+    pip \
+    setuptools \
+    wheel
```

Comment:
Shay says: debian buster needed a newer version of pip to pull in wheels rather than build  from source. Not sure if an upgrade to setuptools is required here.

Reply:
> Not sure if an upgrade to setuptools is required here.

Why would it?

Reply:
I checked on my local branch and looks like you can rip it out-also wheel.

Reply:
Thanks. I wonder why we needed to upgrade pip after the poetry changes, but not before?

Reply:
See 18f91a7ac.

Reply:
Resolved in f0d72acebfd373b6428e8841898e90851a941192